tests/kafkatest/tests/core/consume_bench_test.py

```diff
@@ -68,12 +67,21 @@ def produce_messages(self, topics, max_messages=10000):
         self.logger.debug("Produce workload finished")
 
     @cluster(num_nodes=10)
-    @matrix(topics=[["consume_bench_topic[0-5]"]], metadata_quorum=quorum.all_non_upgrade) # topic subscription
-    @matrix(topics=[["consume_bench_topic[0-5]:[0-4]"]], metadata_quorum=quorum.all_non_upgrade)  # manual topic assignment
-    def test_consume_bench(self, topics, metadata_quorum=quorum.zk):
+    @matrix(
+        topics=[
+            ["consume_bench_topic[0-5]"],
+            ["consume_bench_topic[0-5]:[0-4]"]
+        ],
+        metadata_quorum=quorum.all_non_upgrade,
+        use_new_coordinator=[True, False]
+    ) # topic subscription
+    @skip_if_new_coordinator_and_zk
+    def test_consume_bench(self, topics, metadata_quorum=quorum.zk, use_new_coordinator=False):
         """
         Runs a ConsumeBench workload to consume messages
         """
+        self.kafka.use_new_coordinator = use_new_coordinator
```

Comment:
I see that you only do this here, why that?

Reply:
sry the last commit had these